src/runner/backgroundlayer.rs

```diff
@@ -9,7 +9,7 @@ pub struct BackgroundLayerPlugin;
 impl Plugin for BackgroundLayerPlugin {
     fn build(&self, app: &mut App) {
         app.add_startup_system(spawn_background)
-            .add_system(move_background);
+            .add_system(move_background.system());
```

Comment:
It looks like this was added back? I think we can get rid of `system()` here

Reply:
Yes, it's better to remove the `.system()` suffix, maybe it is a leftover from the 0.5 experience and @cdsupina's tiredness